src/JsonValidation.js

```diff
@@ -208,7 +208,9 @@ var JsonValidators = {
                     }
                 }
                 if (s.length > 0) {
-                    report.addError("OBJECT_ADDITIONAL_PROPERTIES", [s], null, schema);
+                    for (const i of s) {
+                        report.addError("OBJECT_ADDITIONAL_PROPERTIES", [i], null, schema);
```

Comment:
i think missing required properties also reports errrors in an aggregated way

Reply:
@vladbarosan do we have any case where we would like to suppress missing required property?